exercises/pangram/PangramExample.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+// Needed to strip diacritics
+
+struct Pangram {
+
+    static func isPangram(_ text: String) -> Bool {
```

Comment:
I guess my personal preference is not to do this, at least for the purposes of this project. It seems cleaner to me to write a separate Class or Struct per exercise, rather than extending the built-in types.